metabase/metabase
# Add global styles to storybook, fix default fonts display

### Description

This adds missing global styles that we have in the Metabase app, which are needed to make sure stories look like in the main web app.
This also fixes default fonts display - Lato should be displayed now.

### How to verify

Run storybook
Check different stories - e.g. https://storybook.metabase.com/?path=/story/viz-linechart--default , https://storybook.metabase.com/?path=/story/upsells-pill--default-story

### Demo

<table>
<tr>
<th> Before: </th>
<th> After: </th>
</tr>
<tr>
<td>

<img width="246" alt="Screenshot 2024-06-21 at 19 41 57" src="https://github.com/metabase/metabase/assets/7983744/2d07d4a2-8e35-4618-9361-1095144bbd90">


</td>
<td>

<img width="246" alt="Screenshot 2024-06-21 at 19 42 07" src="https://github.com/metabase/metabase/assets/7983744/cc01521b-91f3-4852-b259-6f23725274c2">



</td>
</tr>
<tr>
<td>

<img width="800" alt="Screenshot 2024-06-21 at 19 42 31" src="https://github.com/metabase/metabase/assets/7983744/efbc4e1a-b284-4c89-8912-167df1cc46ba">



</td>
<td>

<img width="800" alt="Screenshot 2024-06-21 at 19 44 06" src="https://github.com/metabase/metabase/assets/7983744/a7944ae0-e2bf-425b-90e1-7b374c0dca3d">




</td>
</tr>
</table>
